alerting/build.gradle

```diff
@@ -16,7 +16,7 @@ apply plugin: 'jacoco'
 
 def usingRemoteCluster = System.properties.containsKey('tests.rest.cluster') || System.properties.containsKey('tests.cluster')
 def usingMultiNode = project.properties.containsKey('numNodes')
-String bwcVersion = "2.6.0.0"
+String bwcVersion = "2.10.0.0"
```

Comment:
can we set this dynamically by building a new string from "opensearch_version"

Reply:
api "org.opensearch.plugin:percolator-client:${opensearch_version}" is using opensearch_version in this file.. so we can use that itself i guess by removing -SNAPSHOT if required. plz check

Reply:
No since this is the BWC version, not the current opensearch version. We wont know the latest minor version (eg. 2.10) from the last major version (eg. 2.x).